### PR TITLE
Replace Delorean with dateutil, as dateutil is brought in by botocore

### PR DESCRIPTION
**Pull request by funkybob**
No point depending on two libs that provide the same services.

### CODE REVIEW

**Comment by Dunedan** [comment]
I'd really appreciate seeing this being merged, as `Delorean` and its dependencies pull in additional ~25MB of unnecessary packages.

**Comment by funkybob** [comment]
@brandond yeah, that's why I made the change - great lib, terribly weighty deps...

File: setup.py [hunk 1]
```diff
@@ -12,7 +12,6 @@
     sys.exit()
 
 install_requires = [
-    'Delorean',
```
Comment (garrettheel) [inline setup.py]: Can you declare `python-dateutil` here to make it clear we're relying on it?

**Comment by garrettheel** [comment]
Tests still use `Delorean`. Can you replace these with some that sanity check `dateutil` instead?

File: setup.py [hunk 2]
```diff
@@ -12,9 +12,9 @@
...
     'six',
-    'botocore>=1.0.0'
+    'botocore>=1.0.0',
+    'python-dateutil>=2.6.0',
```
Comment (garrettheel) [inline setup.py]: Can you add `<3.0.0` here to match botocore also?

**Comment by garrettheel** [comment]
Thanks for the updates! One more small comment and then this should be good to go.

**Comment by garrettheel** [comment]
Thanks for contributing

**Comment by geeknam** [comment]
Any chance of releasing a pypi version that includes this change? This would make my packaged lambdas much smaller.

**Pull request merged**